torchtitan/experiments/deepseek_v3/generate.py

```diff
@@ -118,14 +121,27 @@ def generate(mesh: DeviceMesh, messages: list[dict], n_tokens: int = 10):
             x[:, next_idx] = next_token
             next_idx += 1
     if rank == 0:
-        print(tokenizer.decode(x[0]))
+        output = tokenizer.decode(x[0])
+        # Clean up the output by removing special tokens
+        output = output.replace("<｜begin▁of▁sentence｜>", "")
```

Comment:
Maybe use tokenizer.bos/eos tokens directly? Not 100% sure but I think these may be different between v2, v3.

Reply:
Yes, agree with this.  Just wasn't sure where to get the tokenization ids for this.

Reply:
fixed - verified that it's as easy as tokenizer.eos and .bos.  
Thanks for flagging this!